azure-pipelines.release.yml

```diff
@@ -0,0 +1,64 @@
+trigger:
+  branches:
+    include:
+      - release-*
+
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: AzurePipelines-EO
+      image: 1ESPT-Mariner2.0
+      os: linux
```

Comment:
Is this the pipeline you want to be running on, or was this the default configuration from the migration tool?

Reply:
I wrote this pipeline by hand; this was just recommended in the docs.